libres/lib/include/ert/analysis/ies/ies_enkf.hpp

```diff
@@ -23,13 +23,30 @@
 #include <ert/util/rng.hpp>
 
 #include <ert/res_util/matrix.hpp>
+#include <ert/analysis/ies/ies_enkf_data.hpp>
+
+#ifdef __cplusplus
+extern "C" {
```

Comment:
Is this necessary now that we are using C++?
My understanding is that this is to prevent name-mangling and has to be used if we want to call the functions via `cwrap`.

Reply:
My previous experience is that removing these very quickly ripple out of control - but I agree it is a good goal and I will try to remove them.